src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceContractTests.cs

```diff
@@ -19,25 +19,35 @@ public static class ServiceContractTests
     public static void DefaultSettings_Echo_RoundTrips_String_Buffered()
```

Comment:
Maybe these could be consolidated into a [Theory] test? It looks like much of this is repeated code.